src/components/Stack/__stories__/AlignItems.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import { Template } from './Template.stories'
+import { coloredChildren } from './helper'
+
+export const AlignItems = Template.bind({})
+
+AlignItems.parameters = {
+  docs: {
+    storyDescription:
+      'prop `alignItems` support every value of css property `align-items`',
```

Comment:
I think we should add `The` at the beginning of each description:

```suggestion
      'The prop `alignItems` supports every value of css property `align-items`',
```